### PR TITLE
[Doc Improvement][Update link-unfurling]

### PR DESCRIPTION
**Pull request by v-ypalikila**
Added changes.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/492e1bdf431ff12409b0230dfb9dafdd718f6cc3/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ce59211a-2585-4372-ae5a-e7f8fe1b0eaf) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a90bbd564e08c028c6ccc3f3d63bca08393551ae/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6249649d-9ec5-4a1f-91c9-a0be6096ea64) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a90bbd564e08c028c6ccc3f3d63bca08393551ae/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 94 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e00b8956-e091-480e-90b1-7575bccb8f28) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/95491d8b042b25a7064b54990efc195a644ca701/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7888b607-ee11-409d-a4ba-8ad68e269a6e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/95491d8b042b25a7064b54990efc195a644ca701/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 94 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5709defc-df96-4202-a2fc-0c0c45e41f57) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5fcd2b0127548917725627a40ff1125df836ce1c/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/80356ffd-63c3-4fb4-9ac5-434cd0c52e31) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5fcd2b0127548917725627a40ff1125df836ce1c/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 94 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4d2539fa-04f2-469e-af71-b8d0edd140a6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/678eed7ef8dcd5cf211b2d7bd51d01beb85b3fda/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5e007845-ed11-4791-8c16-2b796f409c03) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/678eed7ef8dcd5cf211b2d7bd51d01beb85b3fda/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/56027939-0854-41c4-90de-8d2c09bc2456) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1f10e7c4e5927c7c63f5f84b5bb236844801eda5/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/191be653-a447-439f-b0ed-589ea8c33afc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1f10e7c4e5927c7c63f5f84b5bb236844801eda5/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b50b937a-51aa-49b8-82aa-83d14b6cefd2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8aaa59df8155e71b14f50e2bf583a3ed30196630/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 92 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2b217458-1176-4ef9-a9c8-cbb3bea15da4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8aaa59df8155e71b14f50e2bf583a3ed30196630/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/460e0f56-e852-4c84-adb5-8ba657ef3400) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

File: msteams-platform/messaging-extensions/how-to/link-unfurling.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ This document guides you on how to add link unfurling to your app manifest using
 > * Currently, link unfurling is not supported on Mobile clients.
 > * The link unfurling result is cached for 30 minutes.
 
-The Azure DevOps messaging extension uses link unfurling to look for URLs pasted into the compose message area pointing to a work item. In the following image, a user has pasted a URL for a work item in Azure DevOps, which the messaging extension has resolved into a card:
+When a URL is pasted in the compose message area, the web service **Unfurls** the URL into a detailed card.
+
+ The following image displays link unfurling feature when a link is pasted in messaging extension:
```
Comment (surbhigupta12) [inline msteams-platform/messaging-extensions/how-to/link-unfurling.md:21]: Why has this text been updated? The original information is right. Please revert. It doesn't seem to be part of the issue.

File: msteams-platform/messaging-extensions/how-to/link-unfurling.md [hunk 2]
```diff
@@ -64,16 +67,9 @@ For a complete manifest example, see [manifest reference](~/resources/schema/man
...
 ## Handle the `composeExtension/queryLink` invoke
 
-After adding the domain to the app manifest, you must update your web service code to handle the invoke request. Use the received URL to search your service and create a card response. If you respond with more than one card, only the first card response is used.
-
-The following card types are supported:
-
-* [Thumbnail card](~/task-modules-and-cards/cards/cards-reference.md#thumbnail-card)
-* [Hero card](~/task-modules-and-cards/cards/cards-reference.md#hero-card)
-* [Office 365 Connector card](~/task-modules-and-cards/cards/cards-reference.md#office-365-connector-card)
-* [Adaptive Card](~/task-modules-and-cards/cards/cards-reference.md#adaptive-card)
```
Comment (surbhigupta12) [inline msteams-platform/messaging-extensions/how-to/link-unfurling.md:74]: Please revert. Why has this section been deleted. Only line 76 had to be removed.

File: msteams-platform/messaging-extensions/how-to/link-unfurling.md [hunk 3]
```diff
@@ -64,16 +67,9 @@ For a complete manifest example, see [manifest reference](~/resources/schema/man
...
+After adding the domain to the app manifest, you must update your web service code to handle the invoke request.
 
-You can display a preview of an Adaptive Card or Office 365 Connector card in the result list using its preview property. The preview property is not necessary if the results are already Hero or Thumbnail cards. If you use the preview attachment, it must be either a Hero or Thumbnail card. If no preview property is specified, the preview of the card fails and nothing is displayed.
+ Use the received URL to search your service and create a card response. If you respond with more than one card, only the first card response is used.
```
Comment (surbhigupta12) [inline msteams-platform/messaging-extensions/how-to/link-unfurling.md:72]: Revert to original

File: msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ ms.localizationpriority: none
 
 [!include[v4-to-v3-SDK-pointer](~/includes/v4-to-v3-pointer-me.md)]
 
-After the user submits the search command, your web service receives a `composeExtension/query` invoke message that contains a `value` object with the search parameters. This invoke is triggered with the following conditions:
+After the user submits the search command, your web service receives a `composeExtension/query` invoke message that contains a `value` object with the search parameters, which is triggered with the following conditions:
```
Comment (surbhigupta12) [inline msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md:13]: Modified sentence is too long. Please revert to original.

File: msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md [hunk 5]
```diff
@@ -73,7 +73,7 @@ The JSON below is shortened to highlight the most relevant sections.
 
 ## Respond to user requests
 
-When the user performs a query, Microsoft Teams issues a synchronous HTTP request to your service. At that point, your code has `5` seconds to provide an HTTP response to the request. During this time, your service can perform additional lookup, or any other business logic needed to serve the request.
+When the user performs a query, Microsoft Teams issues a synchronous HTTP request to your service. At that point, your code has `5` seconds to provide an HTTP response to the request. During this time, your service can do additional lookup, or any other business logic needed to serve the request.
```
Comment (surbhigupta12) [inline msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md:76]: Do is not the right usage. Please revert to original.

File: msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md [hunk 6]
```diff
@@ -99,15 +99,20 @@ To have a better understanding and overview on cards, see [what are cards](~/tas
...
+* Using the `title`, `text`, and `image` properties of the `attachment` when a `preview` property isn't available.
+* Hero or Thumbnail cards don't support Button and tap actions in the preview.
 
-You can display a preview of an Adaptive Card or Office 365 Connector card in the result list using its preview property. The preview property is not necessary if the results are already Hero or Thumbnail cards. If you use the preview attachment, it must be either a Hero or Thumbnail card. If no preview property is specified, the preview of the card fails and nothing is displayed.
+Use `preview` property as Adaptive Card or Office 365 Connector card to display a preview of Adaptive Card or Office 365 Connector card.
+
+Use preview attachment as Hero or Thumbnail card to display a preview of Hero or Thumbnail card.
+
+> [!NOTE]  
+> If the link unfurling doesn't work, check the code for preview property mentioned in the earlier section.
```
Comment (surbhigupta12) [inline msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md:115]: Please rephrase and make use of style guide.

File: msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md [hunk 7]
```diff
@@ -308,7 +313,7 @@ class TeamsMessagingExtensionsSearchBot extends TeamsActivityHandler {
 
 ## Default query
 
-If you set `initialRun` to `true` in the manifest, Microsoft Teams issues a **default** query when the user first opens the messaging extension. Your service can respond to this query with a set of pre-populated results. This is useful when your search command requires authentication or configuration, displaying recently viewed items, favorites, or any other information that is not dependent on user input.
+If you set `initialRun` to `true` in the manifest, Microsoft Teams issues a **default** query when the user first opens the messaging extension. Your service can respond to the query with a set of pre-populated results. This is useful when your search command requires authentication or configuration, displaying recently viewed items, favorites, or any other information that isn't dependent on user input.
```
Comment (surbhigupta12) [inline msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md:316]: Please do not introduce errors. Revert "the" to "this".

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Please incorporate the comments.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8715e054a5bd3c908110ab1932b7ef501d5a0fec/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5d7f377d-8b60-43fe-9997-7f3a250eb0be) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8715e054a5bd3c908110ab1932b7ef501d5a0fec/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1319e79b-af7b-4121-af41-16e3e14baac7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c6295d8630c9f426c50d67850d697d06287caf38/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6ffc23bc-98c2-433a-89cf-4203dff0a479) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c6295d8630c9f426c50d67850d697d06287caf38/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 90 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1f5831cb-8341-41ca-b802-e134cc0a7e52) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/af0760cba56c17922f222de47d94bbff054a57fa/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7debd0aa-09d6-4567-be6c-f8bfe0548348) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/af0760cba56c17922f222de47d94bbff054a57fa/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 90 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/17b81476-669c-42fd-90cf-e051170fb03d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md [hunk 8]
```diff
@@ -99,15 +99,19 @@ To have a better understanding and overview on cards, see [what are cards](~/tas
 
 To learn how to use the thumbnail and hero card types, see [add cards and card actions](~/task-modules-and-cards/cards/cards-actions.md).
 
-For additional information regarding the Office 365 Connector card, see [Using Office 365 Connector cards](~/task-modules-and-cards/cards/cards-reference.md#office-365-connector-card).
+For more information about the Office 365 Connector card, see [Using Office 365 Connector cards](~/task-modules-and-cards/cards/cards-reference.md#office-365-connector-card).
```
Comment (v-habibaumme) [inline msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md:102]: Please revert to additional

File: msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md [hunk 9]
```diff
@@ -99,15 +99,19 @@ To have a better understanding and overview on cards, see [what are cards](~/tas
...
 
-The result list is displayed in the Microsoft Teams UI with a preview of each item. The preview is generated in one of the two ways:
```
Comment (v-habibaumme) [inline msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md:104]: Is there a result list. Please confirm and then make the change

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b2bba431fe89ffa96715b65b8be97e8ee23c1b14/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fd5d024d-605c-4565-ba6e-5f096d44d612) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b2bba431fe89ffa96715b65b8be97e8ee23c1b14/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 89 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d5d6a807-b5e5-4004-9be5-1d4810a60996) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/link-unfurling.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4589d6acdb225c0badf3ab98ebd649885e127320/msteams-platform/messaging-extensions/how-to/link-unfurling.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5b034f1f-6066-47a0-b34e-8e45ead9685c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4589d6acdb225c0badf3ab98ebd649885e127320/msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md) | 88 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/68b28fd8-ce89-4404-aa50-7a8adc69e1d2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Review by juna-gupta — COMMENTED** [review]
https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/search-commands/respond-to-search?tabs=dotnet#response-card-types-and-previews

Card types and Preview card information seems to be moved to a different page now (seems incorrect). Search action command page includes Url unfurling example.

**Pull request merged**